test/cli/cli.test.js

```diff
@@ -32,4 +32,29 @@ describe("cli", () => {
         });
         expect(stdout.trim()).toEqual("hi from foo()");
     });
+
+    it("throws syntax errors once", async () => {
+        let filename = "errors/syntax-error.brs";
+        let stderr = await runWithErrors(filename);
+        expect(stderr.split(filename).length).toEqual(3);
```

Comment:
Mind adding a comment to explain why we expect this to be 3?  It's not very clear that we're checking the number of stack frames by counting the chunks around the executed filename 🤔

Reply:
That's because it is split without removing blank lines so there's an exta entry. I can change that.